Dockerfile

```diff
@@ -1,31 +1,31 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
 RUN apk --no-cache add \
     curl \
     git \
     libc6-compat \
     openssh-client \
-    python \
+    python3 \
```

Comment:
Just to make sure. Did you test this? Are the tools compatible? How about the other Python dependencies: Are those still fine?

Reply:
sorry for the delay, I've run the commands and after my latest change seems to work